From 9108849f974a6513027b2cb80f64e5ee0f22a9fe Mon Sep 17 00:00:00 2001
From: jialv01 <Jianlin.Lv@arm.com>
Date: Wed, 10 Apr 2019 14:23:31 +0800
Subject: [PATCH] pin pynacl to 1.1.2 recent version fail to install

pin pynacl to 1.1.2, Recent versions of pynacl, specifically 1.2.0, 1.2.1, and 1.3.0 fail to install

Signed-off-by: jialv01 <Jianlin.Lv@arm.com>
---
 requirements.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/requirements.txt b/requirements.txt
index 5f2cfd2..004a069 100755
--- a/requirements.txt
+++ b/requirements.txt
@@ -63,6 +63,7 @@ etcd3==0.7.0
 pyparsing==2.2.0
 packaging==17.1
 pexpect==4.6.0
+pynacl==1.1.2
 
 # python-consul>=0.6.1  we need the pre-released version for now, because 0.6.1 does not
 # yet support Twisted. Once this is released, it will be the 0.6.2 version
-- 
2.17.1

